ranger/core/actions.py

```diff
@@ -1591,15 +1591,15 @@ def _recurse_hardlinked_tree(self, source_path, target_path):
                 link(source_path,
                      next_available_filename(target_path))
 
-    def paste(self, overwrite=False, append=False, dest=None):
+    def paste(self, overwrite=False, append=False, dest=None, make_safe_path=None):
```

Comment:
Can't we just assign `get_safe_path` by default rather than `None`? Should simplify some of the other code as well.

Reply:
Yes but at the moment `shutil_generatorized.py` appears to be 'private' (it's only imported in the `generate` function of `CopyLoader`). Should we move `get_safe_path` to a new file?